### DISCUSSION TITLE
DAMASK v3 How to Choose Element Type

### DISCUSSION THREAD

**Discussion by sadiksefaacar** [damask-multiphysics/DAMASK#45] [Q&A]
Hello everyone,

What is the default element type in the grid solver in Damask v3? And how can I change the element type? I have read the Damask v2 page, and there were many different element types identified with ID numbers. What are the element type options in Damask v3? My question was mainly about the FEM solver as I am not really proficient in FFT solvers, is there a similar element type option for FFT solver?

Thanks in advance

SSA

**Comment by eisenlohr** [comment]
The element type is selected by your meshing choice in MSC.Marc.

When using the grid solver, each "element" is a hexahedral cell with a single evaluation point in its center.

**Reply by sadiksefaacar** [reply]
Thank you for your response Dr. Eisenlohr. As I understand it, the spectral solver uses a set of global basis functions defined over the entire solution domain, which are then evaluated at discrete grid points. Regarding the FEM solver within the grid solver framework, is its procedure similar to that of conventional finite element solvers? Additionally, is the concept of a "cell with a single evaluation point" analogous to a linear hexahedral element with reduced integration?

**Reply by eisenlohr** [reply]
Yes, the spectral method at the core of many of the solvers in `DAMASK_grid` uses a set of global basis functions to approximate the solution.

`DAMASK_mesh` is using a FEM approach.

The single evaluation point sits in the center of a cell, similar to where an integration point for reduced integration (with linear shape function) would be.
